suites/upgrade/client-upgrade/firefly-x/basic/2-workload/rbd_cli_import_export.yaml

```diff
@@ -2,6 +2,7 @@ tasks:
 - workunit:
     branch: firefly
     clients:
+      exclude_packages: ['ceph-test', 'ceph-test-dbg']
```

Comment:
this should be in the install.upgrade configuration, not the workunit configuration